tripleo_auto_abandon/auto_abandon.py

```diff
@@ -49,6 +49,17 @@ def load_config():
     CONF(['--config-file', 'auto-abandon.conf'])
 
 
+def _dry_run_msg(url, data):
+    return ("DRY RUN: POST %s DATA: %s" %(url, data))
+
+
+def purty_print(msg):
```

Comment:
heh thanks for merging, was mostly a 'this would be nice and i poked a bit' - was going to say feel free to use and or ignore as you like, thanks